src/main/java/htsjdk/samtools/SAMUtils.java

```diff
@@ -721,15 +730,15 @@ public static long findVirtualOffsetOfFirstRecordInBam(final File bamFile) {
                     refBase += length;
                     break;
                 default:
-                    throw new IllegalStateException("Case statement didn't deal with " + cigarTypeName + " op: " + e.getOperator());
+                    throw new IllegalStateException("Case statement didn't deal with " + cigarTypeName + " op: " + e.getOperator() + "in CIGAR: " + cigar.toString());
```

Comment:
`toString()` is unnecessary here and below since `"text" + obj` will automatically call `obj.toString()`. You can enable a setting in intellij to mark these in the editor and auto-fix them for you.